.github/test-analysis.yml

```diff
@@ -0,0 +1,46 @@
+name: HSF CMS Analysis Test
+
+on:
+  pull_request:
+  push:
+    branches: 
+     - master
+     - main
+
+jobs:
+  test-analysis:
+    runs-on: ubuntu-latest
+    container: rootproject/root:${{ matrix.version }}
+    strategy:
+      matrix:
+        version: [6.22.08-conda, 6.24.06-conda, latest]
```

Comment:
I'd say lest just keep `latest` in here to conserve resources

Reply:
DONE